app/models/derby_event.rb

```diff
@@ -2,11 +2,21 @@ class DerbyEvent < ActiveRecord::Base
   DATE_FORMAT = '%m/%d/%Y'
   TIME_FORMAT = '%l:%M %P'
 
+  RULESETS = %i(mrda wftda jrda made usars rdcl)
```

Comment:
I'm not sure if I would prefer just mashing these into a single field in the db instead of each having their own boolean, I guess presumably you're going to search by them so it's probably worth it. Either way you might be adding / removing them and they are kind of 'magic' strings so I would prefer putting them in the top in a constant like this.